supply-chain/config.toml

```diff
@@ -440,7 +436,7 @@ version = "0.9.6"
 criteria = "safe-to-deploy"
 
 [[exemptions.libc]]
-version = "0.2.133"
+version = "0.2.141"
 criteria = "safe-to-deploy"
```

Comment:
Would it be ok to add an audit for 133 -> 141? I'd hope that libc updates are at least pretty minor

Reply:
It's just (checks) 13 thousand lines of diff... :upside_down_face: . I'll look through it and see if I can come up with something meaningful to say for an audit.

Reply:
Ok, I've now looked through the diff and added an audit. Many of those lines are for platforms Wasmtime doesn't support.

Reply:
Ah ok I didn't realize that the diff would be so large, but your vet makes sense to me.